BE/src/Controllers/UploadInjestControllers.ts

```diff
@@ -15,6 +17,13 @@ export const UploadInjestController = asyncHandler(
         //must calculate all metrics here
         //store everything in db using package model
 
+        // Given the URL, calculate both cost function values. In the GetPackageController,
+        // I will look at the request and only display one of them if they don't want deps.
+        if (body.URL) {
```

Comment:
we will also want to calculate and store the cost if we are given base64 (content part of body)

I'm not sure who should handle saving the package and installing it (I assume whoever works on upload)

However it may be good to put another comment that states these functions must be called with Content and URL both, but a url will need to be provided to them.